mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/dao/DaoInsertMethodGenerator.java

```diff
@@ -83,7 +86,12 @@ public DaoInsertMethodGenerator(
               Insert.class.getSimpleName());
       return Optional.empty();
     }
-
+    VariableElement runtTimeAttributeParam = null;
```

Comment:
I think we should refactor that fragment(to https://github.com/datastax/java-driver/pull/1237/files#diff-2e74e170fe12b9d3b3b34263492d38ddR94) somewhere else and reuse that block of code. 
Maybe the method in `DaoBase`?

Reply:
:+1:,  at least the detection of the parameter can be extracted to a utility method in `DaoBase`.
The `sublist` call can stay in each child class because that's more convenient.